client/view2/view2.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('myApp.view2', ['ngRoute'])
+
+.config(['$routeProvider', function($routeProvider) {
+  $routeProvider.when('/view2', {
+    templateUrl: 'view2/view2.html',
+    controller: 'View2Ctrl'
+  });
+}])
+
+.controller('View2Ctrl', [function() {
```

Comment:
I would suggest you just write controllers instead of views and hook the controller as described above.

This would hence become:

```
angular.module('myApp').controller('somethingController', ['$scope', function($scope){
...
```